lib/college_web/views/person_view.ex

```diff
@@ -0,0 +1,21 @@
+defmodule CollegeWeb.PersonView do
+  use CollegeWeb, :view
+  alias CollegeWeb.StudentView
+  alias CollegeWeb.TeacherView
+  alias CollegeWeb.JsonViewHelper
+
+  @fields [
+    :id,
+    :first_name,
+    :last_name,
+    :email,
+    :date_of_birth,
+    :gender,
+    :address,
+    :number_of_courses
+  ]
+
+  def render("show.json",%{data: data}) do
+    JsonViewHelper.render_json(data, StudentView, @fields)
+  end
```

Comment:
Please return the map here, no need to JsonViewHelper to render the json.
(Please update all the views to this format)

%{
id: person.id,
first_name: person.first_name,
....
}